.github/workflows/jdk-early-access-build.yml

```diff
@@ -0,0 +1,98 @@
+name: Quarkus Early Access JDK Build
+
+on:
+  schedule:
+    # midnight every weekday + saturday
+    - cron:  '0 0 * * 1-6'
+  workflow_dispatch:
+    inputs:
+      jdkVersion:
+        description: 'JDK version'
+        required: true
+        default: '16-ea'
+
+env:
+  JDK_VERSION: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.jdkVersion || '16-ea' }}
+  # Workaround testsuite locale issue
+  LANG: en_US.UTF-8
+  JVM_TEST_MAVEN_OPTS: "-e -B --settings .github/mvn-settings.xml -Dtest-containers -Dstart-containers -Dformat.skip -DskipDocs"
```

Comment:
I copied this from the CI workflow. Not sure about `-DskipDocs`.

I also copied those DB_* variables. I haven't tested whether those are actually needed.

Reply:
I had forgotten to push something, so I also removed `-DskipDocs`.